balboa-agent/src/main/scala/com/socrata/balboa/agent/Config.scala

```diff
@@ -29,6 +36,18 @@ trait Config extends JMSClientConfig {
 
   def initialDelay(defaultDelay: Long = 0): Long = Configuration.get().getLong(Keys.InitialDelay, defaultDelay)
 
+  def httpOrMQ(defaultMethod: HttpOrMq = HttpOrMq.HTTP): HttpOrMq =
+    Configuration.get().getString(Keys.HttpOrMq, defaultMethod.toString) match {
+      case "HTTP" => HttpOrMq.HTTP
+      case "MQ" => HttpOrMq.MQ
+      case _ => defaultMethod
+    }
+
+  def balboaHttpUrl(defaultUrl: String): String =
```

Comment:
Yeah, I wasn't sure what the future use cases might be for this API, so I decided to just mirror the behavior of the other conf getters. I will change it to return an option.